pyradium/templates/base/pyradium_dialog.css

```diff
@@ -0,0 +1,41 @@
+dialog {
+	border: none;
+	border-radius: 10px;
+	padding: 20px;
+	border: 15px solid #3498db;
+}
+::backdrop {
+	background: rgba(0, 0, 0, 0.45);
+}
+
+dialog .header {
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	flex-flow: row nowrap;
+}
+dialog .close {
+	font-size: 32px;
+	padding: 0;
+	margin: 0;
+	border: none;
+	background-color: inherit;
```

Comment:
Is background-color inherit not the default? I'm not sure. If it is, please omit.

Reply:
No, because this is a hTmL `<button />` element, some default background color is set by the browser hence this line of code is quite important.